src/DETHRACE/common/car.c

```diff
@@ -1469,62 +1437,50 @@ void ControlCar4(tCar_spec* c, br_scalar dt) {
     LOG_TRACE("(%p, %f)", c, dt);
 
     if (c->keys.left) {
-        if (c->turn_speed < 0.0) {
-            c->turn_speed = 0.0;
+        if (c->turn_speed < 0.f) {
+            c->turn_speed = 0.f;
         }
-        if (c->velocity_car_space.v[2] <= 0.0) {
-            if ((c->curvature < 0.0 || c->omega.v[1] < -0.001) && c->turn_speed == 0.0) {
-                c->turn_speed = 0.050000001 / (BrVector3Length(&c->v) + 5.0) * (dt * 25.0) * 4.0 / 2.0 * 0.5;
-                if (c->omega.v[1] < -0.01) {
-                    c->turn_speed = c->turn_speed - dt * 0.01 / 0.04 / 2.0 * c->omega.v[1] * 2.0;
-                }
-            } else {
-                c->turn_speed = 0.050000001 / (BrVector3Length(&c->v) + 5.0) * (dt * 25.0) / 2.0 * 0.5 + c->turn_speed;
-            }
+        if (c->velocity_car_space.v[2] > 0.f) {
+            c->turn_speed += dt * 0.01f / (harness_game_config.physics_step_time / 1000.f) / 2.f * 2.f;
```

Comment:
I don't believe the 0.04 constant is related to physics step. Lets keep it disconnected for now. 

```suggestion
            c->turn_speed += dt * 0.01f / 0.04f / 2.f * 2.f;
```